contrib/elasticsearch2/values.yaml

```diff
@@ -0,0 +1,55 @@
+# Default values for elasticsearch2.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+storage:
+  accessModes: [ "ReadWriteOnce" ]
+  capacity: 1Gi
+
+image:
+  repository: elasticsearch
+  tag: 2.4.6
+  pullPolicy: IfNotPresent
+
+# Whether we should create an ingress rule for the elastichq dashboard
+ingress:
+  enabled: true
```

Comment:
maybe default to false and host=elasticsearch.localhost

Reply:
Changed defaults to `ingress.enabled=false` and `host=elasticsearch.localhost`